build-challenge/tsconfig.json

```diff
@@ -20,7 +20,8 @@
        "rootDir": ".",
        "baseUrl":"behavior_packs/",
        "listFiles":false,
-       "noEmitHelpers":true
+       "noEmitHelpers":true,
+       "skipLibCheck": true
```

Comment:
I googled it but a bit confused - skipLibCheck looks like it skips checks to ensure included types are not checked. Sometimes it's used to get by build errors but generally it seems like it's frowned upon and makes your types less safe. I could see a performance improvement from this but how does it decrease our number of dependencies?

Reply:
I think this should be a temporal work around only to fix the `infinity` enchantment issue.
Dependency reduction is because of core build tasks changes. It is taking latest version.

Reply:
Oh nice!